src/main/java/com/katalon/plugin/testrail/TestRailPreferencePage.java

```diff
@@ -41,6 +41,12 @@
     private Label lblConnectionStatus;
 
     private Thread thread;
+    
+    private boolean initialized = false;
+    
+    private boolean isInitialized() {
```

Comment:
Please use the default method of DialogPage instead. Sorry for my mistake in the previous comment

```
isControlCreated()
```